airflow/cli/commands/scheduler_command.py

```diff
@@ -17,20 +17,27 @@
 
 """Scheduler command"""
 import signal
+from multiprocessing import Process
+from typing import Optional
 
 import daemon
 from daemon.pidfile import TimeoutPIDLockFile
 
 from airflow import settings
+from airflow.configuration import conf
 from airflow.jobs.scheduler_job import SchedulerJob
 from airflow.utils import cli as cli_utils
 from airflow.utils.cli import process_subdir, setup_locations, setup_logging, sigint_handler, sigquit_handler
+from airflow.utils.serve_logs import serve_logs
 
 
 @cli_utils.action_logging
 def scheduler(args):
     """Starts Airflow Scheduler"""
     print(settings.HEADER)
+    skip_serve_logs = args.skip_serve_logs
```

Comment:
This won't currently be set for scheduler sub-command -- you'll need to add it somewhere in the cli parser.